pymatgen/core/structure.py

```diff
@@ -1234,7 +1251,7 @@ def from_spacegroup(
         coords_are_cartesian: bool = False,
         tol: float = 1e-5,
         labels: Sequence[str | None] | None = None,
-    ) -> IStructure | Structure:
```

Comment:
Noticed a lot of cases where `IStructure | Structure` is used where there should be `IStructure` (similar for `IMolecule`). Need to look closer to make sure this is not intended.